include/ccf/pal/attestation.h

```diff
@@ -63,12 +63,12 @@ namespace ccf::pal
     auto quote =
       *reinterpret_cast<const snp::Attestation*>(quote_info.quote.data());
 
-    if (quote.version != snp::attestation_version)
+    if (quote.version < snp::minimum_attestation_version)
```

Comment:
Side effect of moving region+SKU size - we now get v3 attestations.

Reply:
That seems like a thing that would be best split into its own PR, with changelog, and backported to 5.0.13.